config/mosquitto.conf

```diff
@@ -1,7 +1,8 @@
+listener 1883
 persistence true
 persistence_location /mosquitto/data/
 log_dest file /mosquitto/log/mosquitto.log
 
 ## Authentication ##
 # allow_anonymous false
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it’s relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->

**❓ question(non-blocking):** Not sure, but it seems that since version 2 we also have to set `allow_anonymous` to `true` explicitely... No?

Reply:
```
## Authentication ##
# By default, Mosquitto >=2.0 allows only authenticated connections. Change to true to enable anonymous connections.
allow_anonymous false
```

like this?
Maybe we should add to the documentation (or a cmd/shell file) on how to update the password?

Reply:
> ```
> ## Authentication ##
> # By default, Mosquitto >=2.0 allows only authenticated connections. Change to true to enable anonymous connections.
> allow_anonymous false
> ```
> 
> like this? Maybe we should add to the documentation (or a cmd/shell file) on how to update the password?

Yes, this seems great!
About password change, I wrote [this](https://github.com/vvatelot/mosquitto-docker-compose#change-user-password--create-a-new-user) in the readme, but maybe it should be more explicit?